src/Database/Driver/OracleBase.php

```diff
@@ -290,8 +290,9 @@ public function convertPositionalToNamedPlaceholders($query)
      */
     public function lastInsertId(?string $table = null, ?string $column = null)
     {
+        [$schema, $table] = explode('.', $table);
```

Comment:
This is a breaking change if the passed in `$table` is only the tablename and not `schema.table`.

Reply:
This is fixed by checking if table is set, and if its not then setting table to schema then schema to null.